Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -182,7 +192,7 @@ exec "$SHELL"
    > [!WARNING]
    > Be careful with using graphical editors like Notepad in Windows. These can add carriage returns (`\r`) that confuse our Linux-based development tools. Instead, we recommend using editors designed for programming or command-line text editors.
 
-8. Create a virtual environment for oppia by adding file named `.envrc` into the parent folder of the oppia repository
+8. Create a virtual environment for oppia by adding file named `.envrc` into the parent folder of the oppia repository, which is `opensource/` in this case.
```

Comment:
Create a virtual environment for Oppia by adding an `.envrc` file into the parent folder of the oppia repository (`opensource/` in this case). The file should have the following content: